src/java/com/google/cloud/bigquery/dwhassessment/extractiontool/subcommand/ExtractSubcommand.java

```diff
@@ -250,6 +250,15 @@ void setOutputPath(String pathString) {
       defaultValue = "NORMAL")
   private RunMode mode;
 
+  @Option(
+      names = "--need-querytext",
```

Comment:
Why did you choose "need"? Did you consider maybe "extract" or "include/exclude"?

Reply:
Per our offline discussion:
- Names like "get" and "extract", if used for accessing the AutoValue value, may be confused by the reader as a method that actually performs the extraction.
- "include" would work fine although "need" sounds more concise; we both like "exclude" in this case, but to avoid running into conflict with internal java style guides of only using positive/neutral boolean flag names, we have to pass on that :-(
- both "include" and "need" work nicely as an negatable flag as Christian suggested.